--- basic680.asm	2026-04-14 22:22:29
+++ c68basic.asm	2026-04-14 22:42:44
@@ -1,5 +1,12 @@
+; c68basic.asm
+
 ; Loaded Motorola S file basic680.mot
 
+; monitor routines - C68 mikbug2 addresses
+
+INCH    EQU     $E178
+OUTCH   EQU     $E175
+
 ; Missing referenced labels:
 
 M17D8   EQU     $17D8
@@ -73,9 +80,7 @@
 MF002   EQU     $F002
 MF600   EQU     $F600
 MFC18   EQU     $FC18
-INCH    EQU     $FF00
 POLCAT  EQU     $FF24
-OUTCH   EQU     $FF81
 MFFFF   EQU     $FFFF
 
 ;****************************************************
